scalatest/src/main/scala/org/scalatest/refspec/RefSpecLike.scala

```diff
@@ -65,7 +65,7 @@ trait RefSpecLike extends TestSuite with Informing with Notifying with Alerting
         
         def getMethodTags(o: AnyRef, methodName: String) =
           for {
-            a <- getMethod(o, methodName).getDeclaredAnnotations
+            a <- getMethod(o, methodName).getDeclaredAnnotations.map(_.nn)
```

Comment:
**Java method returns array**
Java method returns array of type `Array[Null | T]`. We need to map it to `Array[T]`.